EasyPost/_base/ApiVersion.cs

```diff
@@ -7,6 +7,7 @@ namespace EasyPost._base
     public class ApiVersion : ValueEnum
     {
         public static readonly ApiVersion Beta = new(2, "beta");
+        public static readonly ApiVersion V2 = new(1, "v2");
```

Comment:
### Author's Note

This was added initially purely for documentation purposes, so V2 parameters can reference the V2 API version in their docstrings.

However, this also offers an alternative/future-prep for eventual API versioning. Users can select Current or V2, they both (currently) do the same thing.